thirdweb-dev/js
# [RN] Adds switchToActiveChain to the ConnectWallet button

## Problem solved

Adds switchToActiveChain prop to the ConnectWallet button.

When set to true it will show a "Switch Network" button if the wallet
is connected to a different chain than the `activeChain` provided in `ThirdwebProvider`

## Changes made

Adds new prop: `switchToActiveChain`

## How to test

- [ ] Automated tests: link to unit test file
- [x] Manual tests: step by step instructions on how to test
1. Set switchToActiveChain to true
2. Connect to a wallet with an activeChain different than the activeChain in the ThirdwebProvider. 
3. A Switch Network button should render to switch the chain to the provider's activeChain prop
